doc/architecture/decisions/0002-git-commit-messages.md

```diff
@@ -34,18 +34,18 @@ Other rules are also important!
 
 In order to automate the release notes/changelog process, we also need to define basic semantics:
 
-| Commit message                                           | Release type                     |
-|----------------------------------------------------------|----------------------------------|
-| fix: Stop the O2 leak dock comportment (DC)              | Patch release                    |
-| feat: Implements dog space travel module (BARK)          | Feature release                  |
-| break: Removes the compase from the control room (DOOM)  | Breaking change release          |
-| docs: Peter and the wolf: Cloud version                  | Documentation only changes       |
-| style: Replace tabs with spaces                          | Changes that not affect the code |
-| test: Add unit test at last                              | Add or correct existing tests    |
+| Commit message                                           | Release type                                            |
+|----------------------------------------------------------|-------------------------------------------------------- |
+| fix: Stop the O2 leak dock comportment (DC)              | Patch release                                           |
+| feat: Implements dog space travel module (BARK)          | Feature release                                         |
+| <type>!: Removes the compase from the control room       | Exclamation after the type mark means a breaking change |
```

Comment:
That gets parsed by the markdown parser as a HTML tag, and is invisible in the preview.  Also, this makes no sense as an example, since `<type>` is unspecified.  We need to rewrite more of this document to be clear.

Reply:
@mook-as table removed since we are relying on the conventional commits document.